hooks/terraform-validate.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -e
+
+# OSX GUI apps do not pick up environment variables the same way as Terminal apps and there are no easy solutions,
+# especially as Apple changes the GUI app behavior every release (see https://stackoverflow.com/q/135688/483528). As a
+# workaround to allow GitHub Desktop to work, add this (hopefully harmless) setting here.
+export PATH=$PATH:/usr/local/bin
+
+for file in "$@"; do
+  terraform validate `dirname $file`
```

Comment:
This will actually run `terraform validate` multiple times in the same module if multiple terraform files were changed in the same module.

Can you modify the script to collect all the potential modules first, and then dedup them prior to looping through them to run validate?

Reply:
Great point, fixed 👍  I guess there same should be done for `terraform fmt`, what do you think?